src/DBConnection.hpp

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#include <string>
+#include <iostream>
```

Comment:
@Morheit , include what you use. I see no usage of iostream, but I see usage of string.

Reply:
@malirod , `#include <iostream>` replaced by `#include <string>`